assets/chef-recipes/cookbooks/public_network/recipes/default.rb

```diff
@@ -0,0 +1,22 @@
+file '/usr/bin/hetznerrouting.sh' do
```

Comment:
Files should be provided from templates or static directory. Do not put them into the contents of the recipe.